go/flags/endtoend/vtcombo.txt

```diff
@@ -404,6 +404,7 @@ Flags:
       --v Level                                                          log level for V logs
   -v, --version                                                          print binary version
       --vmodule vModuleFlag                                              comma-separated list of pattern=N settings for file-filtered logging
+      --vplayer-progress-deadline duration                               At what point, without having been able to successfully replicate a pending batch of events, should we consider the vplayer stalled; producing an error and log message and restarting the player. (default 5m0s)
```

Comment:
Why does this need to be a flag?
I ask because I would like us to be really mindful about introducing new flags. We have too many already.

Reply:
Rest of the implementation LGTM

Reply:
It doesn't have to be a flag — I don't yet have clear guidance on when you might want to change the default value — only the feeling that such a scenario may come up and  being able to use a different value would then be helpful. I could make the 5m deadline static...

Reply:
Let's make it static for now. I was also unable to imagine when you would want to change it.